### PR TITLE
Desktop polish

### LINKED ISSUE #3179: [Desktop] Cmd + H should hide the app

### LINKED ISSUE #3180: [Desktop] Command + [ or ] should navigate forward/backwards

### LINKED ISSUE #3187: [Desktop] Unable to open a new window
Unable to open a new window after closing the current window. Currently you have to quit and reload Spectrum in order for a new window to appear.

### PR DESCRIPTION
**Pull request by brianlovin**
**Status**
- [ ] WIP
- [x] Ready for review
- [ ] Needs testing

**Deploy after merge (delete what needn't be deployed)**
- hyperion (frontend)

- Improves the splash loading view
- Makes navbar and titlebar flush with the window to feel more native ✨
- closes #3187 
- closes #3179 
- closes #3180

### CODE REVIEW

File: desktop/src/splash.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
       html {
         width: 100%;
         height: 100%;
-        background: url('../../public/img/media.png') center center no-repeat;
+        background: url('../../public/img/homescreen-icon-144x144.png') center center no-repeat;
```
Comment (mxstbr) [inline desktop/src/splash.html]: Why are we showing the icon there instead of the full "Spectrum"? Not sure how that's better?

**Review by mxstbr — COMMENTED** [review]
This will need a new desktop app release so I guess this is a great time to test the auto updating!

File: desktop/src/splash.html (hunk 1 above)
Comment (mxstbr) [reply desktop/src/splash.html]: This looks very weird?

screen shot 2018-05-30 at 10 10 32

Not a fan at all, I much preferred the previous version with the Spectrum. Going to revert this.

**Review by mxstbr — CHANGES_REQUESTED** [review]
I assume those buttons aren't where they should be?

screen shot 2018-05-30 at 10 11 10

File: desktop/src/splash.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
       html {
         width: 100%;
         height: 100%;
-        background: url('../../public/img/homescreen-icon-144x144.png') center center no-repeat;
+        background: url('../../public/img/media.png') center center no-repeat;
```
Comment (brianlovin) [inline desktop/src/splash.html]: Why did you revert this?

**Comment by brianlovin** [comment]
And those buttons are where they should be - this solves the problem of removing the native titlebar on macOS. What do you think?

File: desktop/src/splash.html (hunk 2 above)
Comment (mxstbr) [reply desktop/src/splash.html]: Look above in the collapsed comment, I really really do not like that. It looks bad in comparison to showing the `media.png`.

**Comment by mxstbr** [comment]
> And those buttons are where they should be - this solves the problem of removing the native titlebar on macOS. What do you think?

I don't think it looks better, honestly I much prefer the grey bar. I think them being off-center looks pretty crappy.

File: desktop/src/splash.html (hunk 2 above)
Comment (mxstbr) [reply desktop/src/splash.html]: Look at this comparison:

screen shot 2018-05-30 at 10 10 32

screen shot 2018-05-30 at 16 18 20

The second one looks so much more like a start screen to me?

Comment (mxstbr) [reply desktop/src/splash.html]: On a scale from 1 to 10 I'm maybe a 4 on how much I care about this, if you're 10/10 convinced we should go with the first one then I'm fine with it, but at least use the actual app icon and not the homescreen icon for the PWA which is different.

**Comment by brianlovin** [comment]
> I don't think it looks better, honestly I much prefer the grey bar. I think them being off-center looks pretty crappy.

I disagree - the gray bar makes us the app look like a lazily-wrapped website (which it is under the hood). Hiding that bar and merging our UI with native UI feels much cleaner and more app-like. I can play with the button centering, it'll just mean tightening up our app UI, although to be honest the way it is in this branch doesn't look bad to me.

File: desktop/src/splash.html (hunk 2 above)
Comment (brianlovin) [reply desktop/src/splash.html]: I guess I disagree here, too - I think the media version is much clunkier and feels less like someone is opening a native app. Having the PWA icon as a squircle with the shadows.

I'd rather have nothing at all and just let people wait a second or two for startup.

I see other native mac apps do this too: just show nothing during boot.

**Comment by mxstbr** [comment]
I think it looks really really bad. :confused:

Maybe we can just add a 38px black bar atop the navbar where that the buttons then sit in the center of?

**Comment by brianlovin** [comment]
> Maybe we can just add a 38px black bar atop the navbar where that the buttons then sit in the center of?

Sure, I'll play with it

**Comment by brianlovin** [comment]
Here's what this will look like with `hiddenInset` title bar style - just adds a bit of extra padding to the traffic lights
screenshot 2018-05-30 08 01 23

Versus how things look today:
screenshot 2018-05-30 08 02 15

IMO I think the flush version looks and feels much better, more native.

**Comment by brianlovin** [comment]
screenshot 2018-05-30 08 13 27

Here it is tidied up and centered.

**Comment by mxstbr** [comment]
I much much much prefer the centered version, that looks like an actual native app!

**Comment by brianlovin** [comment]
Sounds good - those changes are pushed!

Anything else here @mxstbr ? Thanks for the feedback so far

**Review by mxstbr — APPROVED** [review]

**Pull request merged**